tutorials/tutorial.ipynb

```diff
@@ -0,0 +1,1492 @@
+{
```

Comment:
<p>Same with pairing, try and be as explicit as possible, how do I interpret my pairing results.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/solvation-analysis/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Added a bit more detail here</p>

Reply:
Same as with `Coordination` could you also add some detail in the docs?

Reply:
Good call, updates docs for `Coordination` and `Pairing`